python/fledge/plugins/south/mqtt_sparkplug/mqtt_sparkplug.py

```diff
@@ -297,13 +297,17 @@ def on_message(self, client, userdata, msg):
             device_readings = {}
             for metric in sparkplug_payload.metrics:
                 value = "Unknown"
+                data_type = metric.datatype
```

Comment:
Should we check has field data_type? 

If missing is that an error and should be logged?